contracts/apps/disputable/IAgreement.sol

```diff
@@ -12,6 +12,7 @@ import "../../lib/arbitration/IArbitrable.sol";
 contract IAgreement is IArbitrable, IACLOracle {
 
     event Signed(address indexed signer, uint256 settingId);
+    event TransactionFeesOracleSet(address transactionFeesOracle);
```

Comment:
I think we can remove this event and leverage this as a new part of the agreement settings